contracts/contracts/ChatOracle.sol

```diff
@@ -4,6 +4,96 @@ pragma solidity ^0.8.13;
 // Uncomment this line to use console.log
 // import "hardhat/console.sol";
 
+interface IOracleTypes {
+
+    struct OpenAiRequest {
+        // "gpt-4-turbo-preview" or "gpt-3.5-turbo-1106"
```

Comment:
@kristjanpeterson1 remind me why only 2 options here -- why don't we support every GPT model that OpenAI supports?

Reply:
Good question, can add support if/when needed